lldb/source/Utility/Scalar.cpp

```diff
@@ -746,27 +746,18 @@ Status Scalar::SetValueFromData(const DataExtractor &data,
 bool Scalar::SignExtend(uint32_t sign_bit_pos) {
   const uint32_t max_bit_pos = GetByteSize() * 8;
 
-  if (sign_bit_pos < max_bit_pos) {
-    switch (m_type) {
-    case Scalar::e_void:
-    case Scalar::e_float:
-      return false;
+  assert(sign_bit_pos <= max_bit_pos);
+  if (m_type != Scalar::e_int || sign_bit_pos >= (max_bit_pos - 1)) {
```

Comment:
LLVM's Style guidelines state that single-statement blocks for `if/for/while` should not have braces.